### PR TITLE
Fix empty form info on translations

### LINKED ISSUE #15751: Empty form displayed when there's only an empty translations field
### Describe the Bug

This is very much an edge case and should be related to #14778, but when there's an empty translations field with no fields added to the junction translations collection, the "Empty Form" notice is still appearing:

https://user-images.githubusercontent.com/42867097/192229154-5e47de1a-7325-4da1-8f5c-a86179ae8f6f.mp4

After adding a field to the junction translations collection, this issue doesn't happen anymore.

### To Reproduce

1. Create an `articles` collection
2. Add a Translations field
3. Attempt to create an `articles` item

### Errors Shown

No errors shown

### What version of Directus are you using?

main branch (https://github.com/directus/directus/commit/69797776f1d5937fe82328543fc9c03d5f27878b)

### What version of Node.js are you using?

16.17.1

### What database are you using?

PostgreSQL 13.5

### What browser are you using?

Chrome

### How are you deploying Directus?

running locally in dev build

### PR DESCRIPTION
**Pull request by Nitwel**
## After

grafik

grafik

Fixes #15751

### CODE REVIEW

**Comment by Nitwel** [comment]
We can hide the circle with some css for sure!

**Comment by Nitwel** [comment]
Would look like this.

grafik

grafik

**Comment by benhaynes** [comment]
Looks good!

**Review by br41nslug — APPROVED** [review]
LGTM 👍

File: app/src/interfaces/translations/translations.vue [hunk 1]
```diff
@@ -320,6 +320,15 @@ function useLanguages() {
...
 		margin-top: 32px;
+
+		::v-deep(.info) {
+			position: relative;
+			grid-column: 1 / -1;
+
+			.icon {
+				display: none;
+			}
+		}
```
Comment (rijkvanzanten) [inline app/src/interfaces/translations/translations.vue:331]: LBTM! 

Lets instead use a prop on v-info to enable/disable the icon

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]

File: app/src/interfaces/translations/translations.vue (hunk 1 above)
Comment (rijkvanzanten) [reply app/src/interfaces/translations/translations.vue:331]: I'll take care of this now 👍🏻

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**